Cargo.toml

```diff
@@ -40,6 +40,7 @@ members = [
     "engine-tests",
     "engine-transactions",
     "engine-types",
+    "engine-workspace",
```

Comment:
This is now going to replace https://github.com/aurora-is-near/aurora-workspace right?

Reply:
Yes. But this, internal crate is related to the engine only. Also, we have plans to continue working on the aurora-workspace by making it less dependable on the aurora-engine-types and more undependable.

Reply:
Why duplicate the two? Can't the engine use aurora-workspace?

Reply:
No. It can't. The reason is the issue related to the usage of `aurora-engine-types` from different sources. In the tests, we use the local path to the crate, but in the `aurora-workspace`, we use it from the git, and so we've got different versions of types in the engine tests.

Reply:
Would it be possible for engine tests to use engine types defined in aurora workspace? That way we also ensure the workspace is kept up to date. Just an idea.

Reply:
It's not so easy. I thought about this. But the problem is that we use these types further in the code in functions whose parameters are from the local crate.